controllers/topics.controller.js

```diff
@@ -0,0 +1,10 @@
+const { selectTopics } = require("../models/topics.model");
+
+function getTopics(req, res, next) {
+  selectTopics().then(({ rows }) => {
+    const topics = rows;
```

Comment:
You could skip the extra variable by just adding `topics` in the argument of the `.then` without destructuring. So `selectTopics().then((topics) => {
   res.status(200).send ({topics})
})`